### DISCUSSION TITLE
DBC file for general car and specific model

### DISCUSSION THREAD

**Discussion by gaucho1978** [collin80/SavvyCAN#798] [Q&A]
Where can I find a ready to use dbc file suitable for my car? 

First of all congratulations for this amazing tool. I'm using it with canable+ SLCAN firmware.
I would like to use a dbc file to decode messages of my Alfa Romeo Giulia and to identify start button press and some other messages. 
I know I can sniff and try some methods for a reverse engineering of the protocol but it's not easy, and sometime it's not possible.
 Where can I find a ready to use dbc file suitable for my car?

**Comment by collin80** [comment]
The manufacturers tend not to be super motivated to give away DBC files for their cars. As such, this is a problem everyone has. In general, the best place to go looking for DBC files for a car is OpenDBC: https://github.com/commaai/opendbc

Will you find suitable DBC files there? I don't know. Sometimes cars are supported by DBC files you might not expect. For instance, General Motors cars tended to all use GlobalA so the globala dbc files are the correct ones for those cars.

If you don't find DBC files that match your car there then you would have to use google and see if anyone has already done the work for you and reverse engineered messages. The other, quite common, occurrence is for someone to get dealer tools and either reverse engineer those or use them to find info that then can be published on the internet.
